src/api/hash_get.js

```diff
@@ -0,0 +1,19 @@
+class HashGetHandler {
+  constructor(storageMgr) {
+    this.storageMgr = storageMgr;
+  }
+
+  async handle(event, context, cb) {
+    if (event.pathParameters && event.pathParameters.identity) {
+      let identity = event.pathParameters.identity;
```

Comment:
Need to check if `identity` is a DID or an address. If it is an address need to find the linked DID first